projects/cdk/directives/fullscreen/fullscreen.directive.ts

```diff
@@ -0,0 +1,67 @@
+import {DOCUMENT} from '@angular/common';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ElementRef,
+    EventEmitter,
+    inject,
+    Input,
+    Output,
+    signal,
+    ViewChild,
+    ViewEncapsulation,
+} from '@angular/core';
+import {TuiRoot} from '@taiga-ui/core/components/root';
+
+@Component({
+    standalone: true,
+    selector: '[tuiFullscreen]',
+    imports: [TuiRoot],
+    template: '<tui-root #root><ng-content /></tui-root>',
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        '(document:fullscreenchange)': 'onFullscreenChange($event)',
+    },
+})
+export class TuiFullscreen {
+    @ViewChild('root', {read: ElementRef})
+    private readonly root?: ElementRef<HTMLElement>;
+
+    private readonly doc = inject(DOCUMENT);
+    private readonly state = signal(false);
+
+    @Output()
+    public readonly fullscreenEnabled = new EventEmitter<boolean>();
+
+    @Input('tuiFullscreenOptions')
+    public options?: FullscreenOptions = {navigationUI: 'show'};
```

Comment:
Default is "auto":
https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen#auto

Reply:
fixed